Automattic/WP-Job-Manager
# Design Adjustments for the Licenses page for WPJM addons

This PR is the result of a call that I had with @Luchadores regarding possible improvements/adjustments based on the work in #2397. It contains some small design fixes and adjustments for the WPJM Addons License page.


### Changes proposed in this Pull Request

* Adjust "Inactive" section not being moved correctly to align vertically more with the Search input;
* Remove notice style from the message "Lost your license key" in the bottom of the page;
* Changed checkmark icon to match the one from Gutenberg;
* Added  `aria-hidden` to the logo of the product, which isn't an interactive element anyway; (well, this isn't a design change, but I included here anyway)
* Added red border color to the licence field when there's a notice showing an error;
* Changed notices background from white to the color that Gutenberg uses for each notice type;

### Testing instructions

Verify if the WPJM Addons Licenses page design is more aligned with the one from the Figma design.

Check:

- Both the error and success notices
- If the first section is more vertically aligned with the search input
- If the checkmark for an activated product looks good
- If the border of the field of an addon with an error becomes red

### Screenshot / Video

![image](https://user-images.githubusercontent.com/529864/231921508-d4288758-6850-42c5-9d2d-14ec778f3684.png)

![image](https://user-images.githubusercontent.com/529864/231921167-66fd212d-9bbb-4fbd-91cf-90fa39f02e42.png)

![image](https://user-images.githubusercontent.com/529864/231921282-e95242f6-e02a-4d0b-bea7-0bff2d7536ae.png)

![image](https://user-images.githubusercontent.com/529864/231921393-efd4d891-dfb4-4433-90ff-6794273b7438.png)
